PayItForward.Web/Controllers/HomeController.cs

```diff
@@ -50,9 +60,30 @@ public IActionResult Index(int id, string categoryName, string search = "")
             return this.View(resultModel);
         }
 
-        public IActionResult About()
+        [Authorize]
+        public IActionResult MyStories()
         {
-            return this.View();
+            var stories = this.storiesService.Stories()
+                .Where(story => story.User.Id == this.httpaccessor.HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value)
+                .ToList();
+
+            var resultModel = new MyStoriesViewModel()
```

Comment:
You can use the basic story view model, you dont need new model here because you are using only the Basic Story View Model